packages/react-ui-components/src/DateInput/__snapshots__/dateInput.spec.js.snap

```diff
@@ -47,6 +47,13 @@ exports[`<DateInput/> should render correctly. 1`] = `
       dateFormat={true}
       onChange={[Function]}
       open={true}
+      timeConstraints={
+        Object {
+          "minutes": Object {
+            "step": 5,
+          },
+        }
+      }
```

Comment:
How is this change related?

Reply:
It's a fix for broken tests, the tests were broken in master at that point. Now it's possible to just rebase.